Formula/zookeeper.rb

```diff
@@ -38,7 +38,9 @@ def shim_script(target)
 
   def default_zk_env
     <<-EOS.undent
-      export ZOOCFGDIR="#{etc}/zookeeper"
+      if [ -z ${ZOOCFGDIR+x} ]; then
```

Comment:
No idea why that link works locally but fails here, but yeah it sure does fail here. I fixed it.